modules/openapi-generator/src/main/resources/python/model.mustache

```diff
@@ -69,7 +69,7 @@ class {{classname}}(object):
 {{/isNullable}}
 {{^isNullable}}
         if {{name}} is not None:
-            self.{{name}} = {{name}}
+            self.{{name}} = {{name}}  # noqa: E501
```

Comment:
Instead of adding `# noqa: E501`, what about using post file processing hook to format the python code (assuming the code formatter can handle long line nicely)?

Reply:
If we were to change it to multiline we would need to do something like.
self.var_name = (
  var_value)
Which is an unusual way to define a variable, though valid.
Looking at the examples, I think the only line which is too long is: xml_itemp.py:L188:
`            self.prefix_namespace_wrapped_array = prefix_namespace_wrapped_array  # noqa: E501`
So I will just rename this variable to fix it.

@wing328 can you point me to an example of a post processing hook file? I am interested in using that functionality in a future PR. For example I would want to fix def method lines and array assignment lines to take up multiple lines if they are over 80 characters long.

Reply:
You will see the following message when running the Python generator:
```
[main] INFO  o.o.c.languages.PythonClientCodegen - Environment variable PYTHON_POST_PROCESS_FILE not defined so the Python code may not be properly formatted. To define it, try 'export PYTHON_POST_PROCESS_FILE="/usr/local/bin/yapf -i"' (Linux/Mac)
[main] INFO  o.o.c.languages.PythonClientCodegen - NOTE: To enable file post-processing, 'enablePostProcessFile' must be set to `true` (--enable-post-process-file for CLI).
```

Please give it a try and let me know if you need any help.

Reply:
Thank you for that info @wing328. I will use it in a future PR. For this one it doesn't look needed.